doc/coding_standard.md

```diff
@@ -129,13 +129,20 @@ In-code comments should use `//`.
 call();
 ```
 
-For [doxygen](https://www.doxygen.nl/) style comments `/** */` should be used.
+For [doxygen](https://www.doxygen.nl/)-style comments, use `///` for single-line comments
```

Comment:
If we do not allow `/** */` single line comments, are we going to remove these throughout the codebase?

Reply:
I started working on that. I am almost done from with `framework/`...

Reply:
Separate PR I take it?

Reply:
Yes